docs/backend/native_api.ipynb

```diff
@@ -332,7 +332,7 @@
     "tokenizer = AutoTokenizer.from_pretrained(\"Skywork/Skywork-Reward-Llama-3.1-8B-v0.2\")\n",
     "prompts = tokenizer.apply_chat_template(CONVS, tokenize=False)\n",
     "\n",
-    "url = \"http://localhost:30030/classify\"\n",
+    "url = \"http://localhost:30030/encode\"\n",
```

Comment:
this should be classify because it is a reward model

Reply:
I find this in `server.py`:

```
@app.api_route("/encode", methods=["POST", "PUT"])
@time_func_latency
async def classify_request(obj: EmbeddingReqInput, request: Request):
    """Handle a reward model request. Now the arguments and return values are the same as embedding models."""
    try:
        ret = await tokenizer_manager.generate_request(obj, request).__anext__()
        return ret
    except ValueError as e:
        return ORJSONResponse(
            {"error": {"message": str(e)}}, status_code=HTTPStatus.BAD_REQUEST
        )
```

I also feel quite strange. Who changed this?